dotcom-rendering/src/web/components/Card/components/CardWrapper.tsx

```diff
@@ -47,6 +48,39 @@ const cardStyles = (
 		background-color: ${isDynamo ? 'transparent' : palette.background.card};
 	`;
 
+	const decidePaletteBrightness = (thePalette: DCRContainerPalette) => {
+		switch (thePalette) {
+			case 'EventPalette':
+				return `96%`;
+			case 'BreakingPalette':
+				return `85%`;
+			case 'EventAltPalette':
+				return `95%`;
+			case 'InvestigationPalette':
+				return `90%`;
+			case 'LongRunningPalette':
+				return `84%`;
+			case 'LongRunningAltPalette':
+				return `95%`;
+			case 'SombrePalette':
+				return `90%`;
+			case 'SombreAltPalette':
+				return `85%`;
+			default:
+				return `90%`;
+		}
```

Comment:
Is there a world where this is set in [decideContainerOverrides.ts](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/lib/decideContainerOverrides.ts)? 

I'm conscious that when adding snippets like this, we fragment the changes required when a new palette is created - it's now another place you need to know where to go when editing / creating palettes

Reply:
We can't set these values in `decidePalette` (or friends) because that only returns colours (strings) but the hover css is... well, I guess it wouldn't be a colour but it could be a string.... Hmm. I'll have a look 👍